grails-app/services/org/pih/warehouse/replenishment/ReplenishmentService.groovy

```diff
@@ -105,8 +105,9 @@ class ReplenishmentService {
         } else {
             validateReplenishment(order)
         }
-
-        order.save(failOnError: true)
+        if (order.hasErrors() || !order.save(failOnError: true)) {
+            throw new grails.validation.ValidationException("Invalid order", order.errors)
```

Comment:
let's import the exception on top of the file, and call it `throw new ValidationException` for better visibility ;)

Reply:
makes sense, will do